data-fusion/s1_flooding_visualisation/script.js

```diff
@@ -0,0 +1,33 @@
+//VERSION=3
+function setup() {
+  return {
+    input: [{
+        datasource: "S2L2A",
+        bands: ["B08", "B03", "B04", "CLM", "CLP"]
+      },
+      {
+        datasource: "S1GRD",
+        bands: ["VV", "VH", "dataMask"]
+      }
+    ],
+    output: { bands: 4 },
+    mosaicking: "ORBIT"
+  };
+}
+
+function toDB(input){
+  return 10 * Math.log(input)/Math.LN10;
+}
+
+function evaluatePixel(sample) {
+  var S1 = sample.S1GRD[0];
```

Comment:
Only the first sample is used in the script. Mayby simple mosaicking makes more sense.

Reply:
multiple dates are used, if you check the EO Browser example you can see the Sentinel-2 image used is a different date to the Sentinel-1 image.

Reply:
Ah, I see the multiple dates are used. I think the different time ranges is part of the payload not the evlascript. Since  both time range you set is a single date, simple mosaicking can do the job in this case. Here's a [link](https://sentinelshare.page.link/qY4a) using simple mosaicking.

Also, remember to remove the commented console.log and return line :)